app/services/katello/candlepin/profiles_uploader.rb

```diff
@@ -0,0 +1,61 @@
+module Katello
+  module Candlepin
+    class ProfilesUploader
+      def self.upload(profile_string:, host: nil, host_id: nil)
```

Comment:
If we're passing a Host every time, is there a need to pass host_id separately?

Reply:
This was more refactoring oddities. Works much better if we just pass `host`.  Updated.